Not able to build from master branch on macos

**Describe the bug**
When doing `RUST_BACKTRACE=full make V=1 NATIVE_FULL_AOT=1 -j$(($(nproc) * 2))` results into error.

**To Reproduce**
Follow the normal build steps.

```shell
git checkout master
./autogen.sh
./configure \
     --prefix="$PWD/nextstep/Emacs.app/Contents/MacOS" \
     --enable-locallisppath="${PWD}/nextstep/Emacs.app/Contents/MacOS" \
     --with-ns \
     --with-imagemagick \
     --with-cairo \
     --with-modules \
     --with-xml2 \
     --with-gnutls \
     --with-json \
     --with-rsvg \
     --with-native-compilation \
     --disable-silent-rules \
     --disable-ns-self-contained \
     --without-dbus \
     --with-xwidgets
RUST_BACKTRACE=full make V=1 NATIVE_FULL_AOT=1 -j$(($(nproc) * 2))
```

**Expected behavior**
successful build

**OS**
`Darwin MacBook-Pro.local 20.4.0 Darwin Kernel Version 20.4.0: Fri Mar  5 01:14:14 PST 2021; root:xnu-7195.101.1~3/RELEASE_X86_64 x86_64`

**Error callstack**
```shell
Running `/Users/jack/emacs-ng/rust_src/remacs-bindings/target/release/remacs-bindings bindings ../rust_src/crates/emacs/src/bindings.rs`
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 2, kind: NotFound, message: "No such file or directory" }', src/main.rs:223:59
stack backtrace:
   0:        0x10d153a54 - std::backtrace_rs::backtrace::libunwind::trace::h79c24a8108eef51e
                               at /rustc/42816d61ead7e46d462df997958ccfd514f8c21c/library/std/src/../../backtrace/src/backtrace/libunwind.rs:90:5
   1:        0x10d153a54 - std::backtrace_rs::backtrace::trace_unsynchronized::hf491b9388f4887f5
                               at /rustc/42816d61ead7e46d462df997958ccfd514f8c21c/library/std/src/../../backtrace/src/backtrace/mod.rs:66:5
   2:        0x10d153a54 - std::sys_common::backtrace::_print_fmt::h5132bce5284c3ec2
                               at /rustc/42816d61ead7e46d462df997958ccfd514f8c21c/library/std/src/sys_common/backtrace.rs:67:5
   3:        0x10d153a54 - <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::hba4e1e451ca8711d
                               at /rustc/42816d61ead7e46d462df997958ccfd514f8c21c/library/std/src/sys_common/backtrace.rs:46:22
   4:        0x10d17469e - core::fmt::write::h7baaf1618474dae0
                               at /rustc/42816d61ead7e46d462df997958ccfd514f8c21c/library/core/src/fmt/mod.rs:1094:17
   5:        0x10d1508da - std::io::Write::write_fmt::hd293de47cc154cdf
                               at /rustc/42816d61ead7e46d462df997958ccfd514f8c21c/library/std/src/io/mod.rs:1580:15
   6:        0x10d1559ef - std::sys_common::backtrace::_print::hb9d4bc7b9e0ae081
                               at /rustc/42816d61ead7e46d462df997958ccfd514f8c21c/library/std/src/sys_common/backtrace.rs:49:5
   7:        0x10d1559ef - std::sys_common::backtrace::print::h82a68481004d7b57
                               at /rustc/42816d61ead7e46d462df997958ccfd514f8c21c/library/std/src/sys_common/backtrace.rs:36:9
   8:        0x10d1559ef - std::panicking::default_hook::{{closure}}::h11b9cc5ac5c4d127
                               at /rustc/42816d61ead7e46d462df997958ccfd514f8c21c/library/std/src/panicking.rs:208:50
   9:        0x10d1554f9 - std::panicking::default_hook::hfe650a460287c541
                               at /rustc/42816d61ead7e46d462df997958ccfd514f8c21c/library/std/src/panicking.rs:225:9
  10:        0x10d155fe5 - std::panicking::rust_panic_with_hook::h5212f5e986dcd234
                               at /rustc/42816d61ead7e46d462df997958ccfd514f8c21c/library/std/src/panicking.rs:591:17
  11:        0x10d155b95 - std::panicking::begin_panic_handler::{{closure}}::hd4a4baba3ac1c064
                               at /rustc/42816d61ead7e46d462df997958ccfd514f8c21c/library/std/src/panicking.rs:497:13
  12:        0x10d153ef8 - std::sys_common::backtrace::__rust_end_short_backtrace::h5a76e76b61bd088d
                               at /rustc/42816d61ead7e46d462df997958ccfd514f8c21c/library/std/src/sys_common/backtrace.rs:141:18
  13:        0x10d155afa - rust_begin_unwind
                               at /rustc/42816d61ead7e46d462df997958ccfd514f8c21c/library/std/src/panicking.rs:493:5
  14:        0x10d18a41f - core::panicking::panic_fmt::h6b7498085d32aaee
                               at /rustc/42816d61ead7e46d462df997958ccfd514f8c21c/library/core/src/panicking.rs:92:14
  15:        0x10d18a505 - core::result::unwrap_failed::h4595b85dafd53acf
                               at /rustc/42816d61ead7e46d462df997958ccfd514f8c21c/library/core/src/result.rs:1355:5
  16:        0x10cfb0907 - remacs_bindings::main::ha44c5990bb58d716
  17:        0x10cfb35ea - std::sys_common::backtrace::__rust_begin_short_backtrace::h0bd938fb402d3b9e
  18:        0x10cfb1fdc - std::rt::lang_start::{{closure}}::h16f1bbc9c2fda9dd
  19:        0x10d156354 - core::ops::function::impls::<impl core::ops::function::FnOnce<A> for &F>::call_once::h88801ec30fa967bc
                               at /rustc/42816d61ead7e46d462df997958ccfd514f8c21c/library/core/src/ops/function.rs:259:13
  20:        0x10d156354 - std::panicking::try::do_call::ha5838b1ed53bb3ce
                               at /rustc/42816d61ead7e46d462df997958ccfd514f8c21c/library/std/src/panicking.rs:379:40
  21:        0x10d156354 - std::panicking::try::h2c2c426e3f3c01a8
                               at /rustc/42816d61ead7e46d462df997958ccfd514f8c21c/library/std/src/panicking.rs:343:19
  22:        0x10d156354 - std::panic::catch_unwind::h383eb7eff10b175f
                               at /rustc/42816d61ead7e46d462df997958ccfd514f8c21c/library/std/src/panic.rs:431:14
  23:        0x10d156354 - std::rt::lang_start_internal::h09b48eb36ffca70d
                               at /rustc/42816d61ead7e46d462df997958ccfd514f8c21c/library/std/src/rt.rs:51:25
  24:        0x10cfb1b99 - _main
make[1]: *** [../rust_src/crates/emacs/src/bindings.rs] Error 101
`

Also running into this - here are my details:

```shell
./configure --enable-rust-debug --with-native-compilation
```
(I also tried this ^ without `--enable-rust-debug`.)

**OS**
`Darwin Macbook-Pro.local 20.5.0 Darwin Kernel Version 20.5.0: Sat May  8 05:10:33 PDT 2021; root:xnu-7195.121.3~9/RELEASE_X86_64 x86_64`

**Error callstack**

```
emacs-ng git:(master) ✗ RUST_BACKTRACE=full make V=1 NATIVE_FULL_AOT=1 -j$(($(nproc) * 2))
/Applications/Xcode.app/Contents/Developer/usr/bin/make -C lib all
/Applications/Xcode.app/Contents/Developer/usr/bin/make -C doc/lispref info
/Applications/Xcode.app/Contents/Developer/usr/bin/make -C doc/lispintro info
/Applications/Xcode.app/Contents/Developer/usr/bin/make -C doc/emacs info
make[1]: Nothing to be done for `info'.
make[1]: Nothing to be done for `info'.
make[1]: Nothing to be done for `info'.
make[1]: Nothing to be done for `all'.
/Applications/Xcode.app/Contents/Developer/usr/bin/make -C lib-src all
make[1]: Nothing to be done for `all'.
/Applications/Xcode.app/Contents/Developer/usr/bin/make -C src VCSWITNESS='$(srcdir)/../.git/logs/HEAD' BIN_DESTDIR=''/Users/mf/code/open-source/emacs-ng/nextstep/Emacs.app/Contents/MacOS/'' \
		 ELN_DESTDIR='/Users/mf/code/open-source/emacs-ng/nextstep/Emacs.app/Contents/Resources/' all
/usr/local/bin/gmkdir -p ../rust_src/crates/emacs/src/
mkdir -p ../rust_src/target/
RUSTFLAGS="" \
	EMACS_CFLAGS="-Demacs  -I. -I. -I../lib -I../lib      -I/usr/local/Cellar/libffi/3.3_3/include -I/usr/local/Cellar/librsvg/2.50.7/include/librsvg-2.0 -I/usr/local/Cellar/gdk-pixbuf/2.42.6/include/gdk-pixbuf-2.0 -I/usr/local/Cellar/libpng/1.6.37/include/libpng16 -I/usr/local/Cellar/libtiff/4.3.0/include -I/usr/local/Cellar/glib/2.68.3/include -I/usr/local/Cellar/cairo/1.16.0_5/include/cairo -I/usr/local/Cellar/glib/2.68.3/include -I/usr/local/Cellar/glib/2.68.3/include/glib-2.0 -I/usr/local/Cellar/glib/2.68.3/lib/glib-2.0/include -I/usr/local/opt/gettext/include -I/usr/local/Cellar/pcre/8.44/include -I/usr/local/Cellar/pixman/0.40.0/include/pixman-1 -I/usr/local/Cellar/fontconfig/2.13.1/include -I/usr/local/opt/freetype/include/freetype2 -I/usr/local/Cellar/libpng/1.6.37/include/libpng16 -I/usr/local/Cellar/libxcb/1.14_1/include -I/usr/local/Cellar/libxrender/0.9.10/include -I/usr/local/Cellar/libxext/1.3.4/include -I/usr/local/Cellar/libx11/1.7.2/include -I/usr/local/Cellar/libxcb/1.14_1/include -I/usr/local/Cellar/libxau/1.0.9/include -I/usr/local/Cellar/libxdmcp/1.1.3/include -I/usr/local/Cellar/xorgproto/2021.4/include  -I/usr/local/Cellar/libpng/1.6.37/include/libpng16 -I/Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk/usr/include/libxml2       -I/usr/local/Cellar/little-cms2/2.12/include       -MMD -MF deps/.d -MP  -I/usr/local/Cellar/jansson/2.13.1/include -I/usr/local/Cellar/gnutls/3.6.16/include -I/usr/local/Cellar/nettle/3.7.3/include -I/usr/local/Cellar/libtasn1/4.17.0/include -I/usr/local/Cellar/libidn2/2.3.1/include -I/usr/local/Cellar/p11-kit/0.24.0/include/p11-kit-1    -pthread -DEMACS_EXTERN_INLINE -Wl,-framework -Wl,Metal" \
	cargo run --release --manifest-path=../rust_src/ng-bindgen/Cargo.toml -- bindings ../rust_src/crates/emacs/src/bindings.rs
/Applications/Xcode.app/Contents/Developer/usr/bin/make -C ../admin/charsets all
echo " " > ../rust_src/target/.rustflags.tmp
/Applications/Xcode.app/Contents/Developer/usr/bin/make -C ../admin/unidata charscript.el
diff -q ../rust_src/target/.rustflags ../rust_src/target/.rustflags.tmp || cp ../rust_src/target/.rustflags.tmp ../rust_src/target/.rustflags
/Applications/Xcode.app/Contents/Developer/usr/bin/make -C ../admin/charsets cp51932.el
/Applications/Xcode.app/Contents/Developer/usr/bin/make -C ../admin/charsets eucjp-ms.el
AWK=awk ./mapconv glibc/ISO-8859-2.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/8859-2.map
AWK=awk ./mapconv glibc/ISO-8859-3.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/8859-3.map
AWK=awk ./mapconv glibc/ISO-8859-4.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/8859-4.map
AWK=awk ./mapconv glibc/ISO-8859-5.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/8859-5.map
AWK=awk ./mapconv glibc/ISO-8859-6.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/8859-6.map
AWK=awk ./mapconv mapfiles/CP932.TXT '/^0x[89A-F][0-9A-F][0-9A-F]/' UNICODE2 ./cp932.awk > ../../etc/charsets/CP932-2BYTE.map
gunzip -c glibc/EUC-JP-MS.gz | awk -f ./eucjp-ms.awk > ../../lisp/international/eucjp-ms.el
AWK=awk ./mapconv glibc/ISO-8859-7.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/8859-7.map
AWK=awk ./mapconv glibc/ISO-8859-8.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/8859-8.map
diff: ../rust_src/target/.rustflags: No such file or directory
AWK=awk ./mapconv glibc/ISO-8859-9.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/8859-9.map
AWK=awk ./mapconv glibc/ISO-8859-10.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/8859-10.map
AWK=awk ./mapconv glibc/ISO-8859-11.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/8859-11.map
rm -f ../rust_src/target/.rustflags.tmp
AWK=awk ./mapconv glibc/ISO-8859-13.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/8859-13.map
awk -f ./blocks.awk < Blocks.txt > ../../lisp/international/charscript.el
AWK=awk ./mapconv glibc/ISO-8859-14.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/8859-14.map
AWK=awk ./mapconv glibc/ISO-8859-15.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/8859-15.map
AWK=awk ./mapconv glibc/ISO-8859-16.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/8859-16.map
AWK=awk ./mapconv glibc/IBM037.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM037.map
AWK=awk ./mapconv glibc/IBM038.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM038.map
AWK=awk ./mapconv glibc/IBM256.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM256.map
AWK=awk ./mapconv glibc/IBM273.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM273.map
AWK=awk ./mapconv glibc/IBM274.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM274.map
AWK=awk ./mapconv glibc/IBM275.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM275.map
AWK=awk ./mapconv glibc/IBM277.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM277.map
AWK=awk ./mapconv glibc/IBM278.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM278.map
   Compiling libc v0.2.95
   Compiling cfg-if v0.1.6
   Compiling version_check v0.1.5
AWK=awk ./mapconv glibc/IBM280.gz '/^<.*[ ] 2/61]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM280.map
   Compiling autocfg v0.1.1
   Compiling cc v1.0.28
   Compiling glob v0.2.11
   Compiling rustc-demangle v0.1.13
AWK=awk ./mapconv glibc/IBM281.gz '/^<.*[ ] 2/61]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM281.map
   Compiling regex v1.1.0
   Compiling lazy_static v1.2.0
   Compiling proc-macro2 v0.4.24
AWK=awk ./mapconv glibc/IBM284.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM284.map
   Compiling ucd-util v0.1.3
   Compiling utf8-ranges v1.0.2
   Compiling unicode-width v0.1.5
   Compiling quick-error v1.2.2
   Compiling unicode-xid v0.1.0
   Compiling termcolor v1.0.4
   Compiling bindgen v0.46.0
   Compiling vec_map v0.8.1
AWK=awk ./mapconv glibc/IBM285.gz '/^<.*[ ] 2/61]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM285.mapnges, cc, vec_map, bindgen(build.rs), regex(build.rs), glob, rustc-demangle, version_check, ...
   Compiling ansi_term v0.11.0
   Compiling strsim v0.7.0
AWK=awk ./mapconv glibc/IBM290.gz '/^<.*[ ] 2/61]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM290.mapnges, cc, vec_map, bindgen(build.rs), ansi_term, regex(build.rs), glob, rustc-demangle, vers...
   Compiling bitflags v1.0.4
   Compiling byteorder v1.2.7
   Compiling scopeguard v0.3.3
AWK=awk ./mapconv glibc/IBM297.gz '/^<.*[ ] 2/61]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM297.mapro2(build.rs), utf8-ranges, cc, vec_map, bindgen(build.rs), ansi_term, regex(build.rs), glob...
AWK=awk ./mapconv glibc/IBM420.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM420.map
AWK=awk ./mapconv glibc/IBM423.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM423.map
awk -f ./cp51932.awk < ../../etc/charsets/CP932-2BYTE.map > ../../lisp/international/cp51932.el
AWK=awk ./mapconv glibc/IBM424.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM424.map
   Compiling peeking_take_while v0.1.2
AWK=awk ./mapconv glibc/IBM437.gz '/^<.*[ ] 2/61]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM437.mapro2(build.rs), utf8-ranges, cc, vec_map, peeking_take_while, bindgen(build.rs), ansi_term, r...
AWK=awk ./mapconv glibc/IBM500.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM500.map
AWK=awk ./mapconv glibc/IBM850.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM850.map
AWK=awk ./mapconv glibc/IBM851.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM851.map
AWK=awk ./mapconv glibc/IBM852.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM852.map
AWK=awk ./mapconv glibc/IBM855.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM855.map
AWK=awk ./mapconv glibc/IBM856.gz '/^<.*[ ] 3/61]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM856.mapmacro2(build.rs), utf8-ranges, cc, vec_map, peeking_take_while, bindgen(build.rs), ansi_term...
   Compiling log v0.4.6
AWK=awk ./mapconv glibc/IBM857.gz '/^<.*[ ] 4/61]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM857.map.rs), utf8-ranges, cc, vec_map, peeking_take_while, bindgen(build.rs), ansi_term, regex(buil...
   Compiling humantime v1.2.0
AWK=awk ./mapconv glibc/IBM860.gz '/^<.*[ ] 5/61]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM860.mapacro2(build.rs), utf8-ranges, cc, vec_map, peeking_take_while, bindgen(build.rs), ansi_term,...
   Compiling memchr v2.1.2
   Compiling thread_local v0.3.6
AWK=awk ./mapconv glibc/IBM861.gz '/^<.*[ ] 7/61]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM861.maproc-macro2(build.rs), utf8-ranges, cc, vec_map, peeking_take_while, bindgen(build.rs), ansi_...
AWK=awk ./mapconv glibc/IBM862.gz '/^<.*[ ] 8/61]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM862.maproc-macro2(build.rs), utf8-ranges, cc, vec_map, bindgen(build.rs), ansi_term, regex(build.rs...
AWK=awk ./mapconv glibc/IBM863.gz '/^<.*[ ] 9/61]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM863.maproc-macro2(build.rs), utf8-ranges, cc, vec_map, bindgen(build.rs), ansi_term, regex(build.rs...
   Compiling textwrap v0.10.0
AWK=awk ./mapconv glibc/IBM864.gz '/^<.*[ ] 9/61]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM864.maproc-macro2(build.rs), utf8-ranges, cc, vec_map, bindgen(build.rs), ansi_term, regex(build.rs...
AWK=awk ./mapconv glibc/IBM865.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM865.map
AWK=awk ./mapconv glibc/IBM866.gz '/^<.*[  10/61]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM866.maproc-macro2(build.rs), utf8-ranges, cc, vec_map, bindgen(build.rs), ansi_term, regex(build.rs...
AWK=awk ./mapconv glibc/IBM868.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM868.map
   Compiling regex-syntax v0.6.4
AWK=awk ./mapconv glibc/IBM869.gz '/^<.*[  10/61]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM869.maproc-macro2(build.rs), utf8-ranges, cc, vec_map, bindgen(build.rs), ansi_term, regex(build.rs...
AWK=awk ./mapconv glibc/IBM870.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM870.map
AWK=awk ./mapconv glibc/IBM871.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM871.map
AWK=awk ./mapconv glibc/IBM874.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM874.map
AWK=awk ./mapconv glibc/IBM875.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM875.map
AWK=awk ./mapconv glibc/IBM880.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM880.map
AWK=awk ./mapconv glibc/IBM891.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM891.map
AWK=awk ./mapconv glibc/IBM903.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM903.map
   Compiling hashbrown v0.1.8
AWK=awk ./mapconv glibc/IBM904.gz '/^<.*[  10/61]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM904.maproc-macro2(build.rs), utf8-ranges, cc, vec_map, bindgen(build.rs), ansi_term, regex(build.rs...
AWK=awk ./mapconv glibc/IBM905.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM905.map
AWK=awk ./mapconv glibc/IBM918.gz '/^<.*[  12/61]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM918.maproc-macro2(build.rs), utf8-ranges, cc, bindgen(build.rs), ansi_term, regex(build.rs), glob, ...
AWK=awk ./mapconv glibc/IBM1004.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM1004.map
AWK=awk ./mapconv glibc/IBM1026.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM1026.map
AWK=awk ./mapconv glibc/IBM1047.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/IBM1047.map
AWK=awk ./mapconv glibc/CP737.gz '/^<.*[ ] 14/61]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/CP737.maputf8-ranges, cc, ansi_term, regex(build.rs), glob, proc-macro2(build), rustc-demangle, log, t...
AWK=awk ./mapconv glibc/CP775.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/CP775.map
AWK=awk ./mapconv glibc/CP1125.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/CP1125.map
AWK=awk ./mapconv glibc/CP1250.gz '/^<.*[  15/61]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/CP1250.maptf8-ranges, cc, ansi_term, regex(build.rs), glob, proc-macro2(build), rustc-demangle, log, t...
AWK=awk ./mapconv glibc/CP1251.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/CP1251.map
AWK=awk ./mapconv glibc/CP1252.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/CP1252.map
AWK=awk ./mapconv glibc/CP1253.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/CP1253.map
AWK=awk ./mapconv glibc/CP1254.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/CP1254.map
AWK=awk ./mapconv glibc/CP1255.gz '/^<.*[  16/61]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/CP1255.map-ranges, cc, ansi_term, regex(build.rs), glob, proc-macro2(build), rustc-demangle, log, thre...
AWK=awk ./mapconv glibc/CP1256.gz '/^<.*[  17/61]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/CP1256.map, cc, ansi_term, regex(build.rs), glob, proc-macro2(build), rustc-demangle, log, thread_loca...
AWK=awk ./mapconv glibc/CP1257.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/CP1257.map
AWK=awk ./mapconv glibc/CP1258.gz '/^<.*[  18/61]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/CP1258.map), utf8-ranges, cc, ansi_term, glob, proc-macro2(build), rustc-demangle, log, thread_local, ...
   Compiling backtrace v0.3.13
AWK=awk ./mapconv glibc/CP10007.gz '/^<.*[ 19/61]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/CP10007.mapce(build.rs), utf8-ranges, cc, ansi_term, glob, proc-macro2(build), rustc-demangle, log, th...
cp mapfiles/CP720.map ../../etc/charsets/CP720.mapglob, regex(build), humantime, libc, memchr(build), backtrace(build.rs), cc, ansi_term, glob, proc-macro2(build), rustc-demangle, log, thread_local, h...
cp mapfiles/CP858.map ../../etc/charsets/CP858.map
AWK=awk ./mapconv glibc/GB2312.gz '/^<.*[ 	]\/x[a-f]/' GLIBC-2-7 ./compact.awk > ../../etc/charsets/GB2312.map
AWK=awk ./mapconv glibc/GBK.gz '/^<.*[ 	]\/x[89a-f]/' GLIBC-2 ./compact.awk > ../../etc/charsets/GBK.map
AWK=awk ./mapconv glibc/GB18030.gz '/^<.*[ 	]\/x..\/x..[ 	]/' GLIBC-2 ./gb180302.awk > ../../etc/charsets/GB180302.map
AWK=awk ./mapconv glibc/BIG5.gz '/^<.*[ 	]\/x[a-f]/' GLIBC-2 > ../../etc/charsets/BIG5.map
AWK=awk ./mapconv glibc/BIG5-HKSCS.gz '/^<.*[ 	]\/x[89a-f].\//' GLIBC-2 ./compact.awk > ../../etc/charsets/BIG5-HKSCS.map
AWK=awk ./mapconv glibc/EUC-TW.gz '/^<.*[ 	]\/x[a-f]/' GLIBC-2-7 ./compact.awk > ../../etc/charsets/CNS-1.map
   Compiling clang-sys v0.26.4
AWK=awk ./mapconv mapfiles/cns2ucsdkw.txt '/^C2/' KANJI-DATABASE ./compact.awk > ../../etc/charsets/CNS-2.maphbrown, textwrap, clang-sys(build.rs), termcolor, strsim, backtrace(build), regex-syntax
AWK=awk ./mapconv mapfiles/cns2ucsdkw.txt '/^C3/' KANJI-DATABASE ./compact.awk > ../../etc/charsets/CNS-3.map
AWK=awk ./mapconv mapfiles/cns2ucsdkw.txt '/^C4/' KANJI-DATABASE ./compact.awk > ../../etc/charsets/CNS-4.map
AWK=awk ./mapconv mapfiles/cns2ucsdkw.txt '/^C5/' KANJI-DATABASE ./compact.awk > ../../etc/charsets/CNS-5.map
AWK=awk ./mapconv mapfiles/cns2ucsdkw.txt '/^C6/' KANJI-DATABASE ./compact.awk > ../../etc/charsets/CNS-6.map
AWK=awk ./mapconv mapfiles/cns2ucsdkw.txt '/^C7/' KANJI-DATABASE ./compact.awk > ../../etc/charsets/CNS-7.map
AWK=awk ./mapconv glibc/EUC-TW.gz '/^<.*\/x8e\/xaf/ s,/x8e/xaf,,' GLIBC-2-7 ./compact.awk > ../../etc/charsets/CNS-F.map
(./mapconv glibc/JIS_X0201.gz '/^<.*[ 	]\/x[0-9]/' GLIBC-1 ./compact.awk && \
	  echo "# Generated by hand" && \
	  echo "0xA1-0xDF 0xFF61" ) > ../../etc/charsets/JISX0201.map
AWK=awk ./mapconv glibc/EUC-JP.gz '/^<.*[ 	]\/x[a-f]/' GLIBC-2-7 \
	  | sed 's/0x2015/0x2014/' > ../../etc/charsets/JISX0208.map
AWK=awk ./mapconv glibc/EUC-JP.gz '/^<.*[ 	]\/x8f/ s,/x8f,,' GLIBC-2-7 ./compact.awk > ../../etc/charsets/JISX0212.map
sed -n -e '/^#/d' -e 's,.*0x\([0-9A-Z]*\)$,/0x0*\1$/d,p' < mapfiles/JISX213A.map > jisx2131-filter
AWK=awk ./mapconv glibc/EUC-JISX0213.gz '/^<.*[ 	]\/x8f/ s,/x8f,,' GLIBC-2-7 > ../../etc/charsets/JISX2132.map
(AWK=awk ./mapconv mapfiles/Uni2JIS '/^[^#].*0-/' YASUOKA ./kuten.awk \
	  | sed -e '/0x2140/s/005C/FF3C/' && \
	  echo '0x3442 0x3D4E' && echo '0x374E 0x25874' && \
	  echo '0x3764 0x28EF6' && echo '0x513D 0x2F80F' && \
	  echo '0x7045 0x9724' ) > ../../etc/charsets/JISC6226.map
cp mapfiles/JISX213A.map ../../etc/charsets/JISX213A.map
AWK=awk ./mapconv glibc/EUC-KR.gz '/^<.*[ 	]\/x[a-f]/' GLIBC-2-7 ./compact.awk > ../../etc/charsets/KSC5601.map
AWK=awk ./mapconv glibc/KSC5636.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/KSC5636.map
AWK=awk ./mapconv glibc/JOHAB.gz '/^<.*[ 	]\/x[89a-f]/' GLIBC-2 ./compact.awk > ../../etc/charsets/JOHAB.map
AWK=awk ./mapconv glibc/KOI-8.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/KOI-8.map
AWK=awk ./mapconv glibc/KOI8-R.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/KOI8-R.map
AWK=awk ./mapconv glibc/KOI8-U.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/KOI8-U.map
AWK=awk ./mapconv glibc/KOI8-T.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/KOI8-T.map
(echo "# Modified from IBM866.map according to the chart at" && \
	  echo "# https://web.archive.org/web/20100131045151/http://www.cyrillic.com/ref/cyrillic/koi-8alt.html" && \
	  echo "# with guesses for the Unicodes of the glyphs." && \
	  sed -e '1 d' \
	     -e '/0xF2/ s/ .*/ 0x2019/' \
	     -e '/0xF3/ s/ .*/ 0x2018/' \
	     -e '/0xF4/ s/ .*/ 0x0301/' \
	     -e '/0xF5/ s/ .*/ 0x0300/' \
	     -e '/0xF6/ s/ .*/ 0x203A/' \
	     -e '/0xF7/ s/ .*/ 0x2039/' \
	     -e '/0xF8/ s/ .*/ 0x2191/' \
	     -e '/0xF9/ s/ .*/ 0x2193/' \
	     -e '/0xFA/ s/ .*/ 0x00B1/' \
	     -e '/0xFB/ s/ .*/ 0x00F7/' < ../../etc/charsets/IBM866.map ) > ../../etc/charsets/ALTERNATIVNYJ.map
AWK=awk ./mapconv mapfiles/bulgarian-mik.txt '1,$' CZYBORRA ./compact.awk > ../../etc/charsets/MIK.map
AWK=awk ./mapconv mapfiles/PTCP154 '/^0x/' IANA ./compact.awk > ../../etc/charsets/PTCP154.map
AWK=awk ./mapconv glibc/TIS-620.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/TIS-620.map
AWK=awk ./mapconv glibc/VISCII.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/VISCII.map
AWK=awk ./mapconv glibc/TCVN5712-1.gz '/^<.*[ 	]\/x[0-9a-f].[ 	]/' GLIBC-1 ./compact.awk > ../../etc/charsets/VSCII.map
AWK=awk ./mapconv glibc/TCVN5712-1.gz '/^<.*[ 	]\/x[2-7a-f].[ 	]/' GLIBC-1 ./compact.awk \
	  | sed 's/0x20-0x7F.*/0x00-0x7F 0x0000/' > ../../etc/charsets/VSCII-2.map
AWK=awk ./mapconv glibc/GEORGIAN-PS.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/KA-PS.map
AWK=awk ./mapconv glibc/GEORGIAN-ACADEMY.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/KA-ACADEMY.map
AWK=awk ./mapconv glibc/HP-ROMAN8.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/HP-ROMAN8.map
AWK=awk ./mapconv glibc/NEXTSTEP.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/NEXTSTEP.map
AWK=awk ./mapconv glibc/MACINTOSH.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/MACINTOSH.map
AWK=awk ./mapconv glibc/EBCDIC-UK.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/EBCDICUK.map
AWK=awk ./mapconv glibc/EBCDIC-US.gz '/^<.*[ 	]\/x/' GLIBC-1 ./compact.awk > ../../etc/charsets/EBCDICUS.map
AWK=awk ./mapconv mapfiles/stdenc.txt '/^[0-9A-Fa-f]/' UNICODE ./compact.awk > ../../etc/charsets/stdenc.map
AWK=awk ./mapconv mapfiles/symbol.txt '/^[0-9A-Fa-f]/' UNICODE ./compact.awk > ../../etc/charsets/symbol.map
AWK=awk ./mapconv glibc/CP949.gz '/^<.*[ 	]\/x[89a-f]/' GLIBC-2 ./compact.awk > ../../etc/charsets/CP949-2BYTE.map
(echo "# Generated from BIG5.map" && \
	  sed -n -e '/0xa140/,/0xc8fe/p' < ../../etc/charsets/BIG5.map | awk -f ./big5.awk ) > ../../etc/charsets/BIG5-1.map
(echo "# Generated from BIG5.map" && \
	  sed -n -e '/0xc940/,$ p' < ../../etc/charsets/BIG5.map | awk -f ./big5.awk ) > ../../etc/charsets/BIG5-2.map
cp mapfiles/MULE-ethiopic.map ../../etc/charsets/MULE-ethiopic.map
cp mapfiles/MULE-ipa.map ../../etc/charsets/MULE-ipa.map
cp mapfiles/MULE-is13194.map ../../etc/charsets/MULE-is13194.map
cp mapfiles/MULE-sisheng.map ../../etc/charsets/MULE-sisheng.map
cp mapfiles/MULE-tibetan.map ../../etc/charsets/MULE-tibetan.mapro2, cc, glob, rustc-demangle, hashbrown, textwrap, clang-sys(build.rs), termcolor, strsim, regex-syntax
cp mapfiles/MULE-lviscii.map ../../etc/charsets/MULE-lviscii.map
cp mapfiles/MULE-uviscii.map ../../etc/charsets/MULE-uviscii.map
AWK=awk ./mapconv glibc/EUC-JISX0213.gz '/^<.*[ 	]\/x[a-f]/' GLIBC-2-7 \
	  | sed -f ./jisx2131-filter \
	  | sed -e 's/0x2015/0x2014/' -e 's/0x2299/0x29BF/' > ../../etc/charsets/JISX2131.map
awk -f ./gb180304.awk < ../../etc/charsets/GB180302.map > ../../etc/charsets/GB180304.map textwrap, clang-sys(build), termcolor, regex-syntax
   Compiling quote v0.6.10
   Compiling atty v0.2.11
   Compiling libloading v0.5.0
echo timestamp > charsets.stamp          ] 38/61: libloading(build.rs), libc, proc-macro2, memchr, atty, glob, quote, bindgen(build), backtrace, regex-syntax
   Compiling clap v2.32.0
   Compiling failure v0.1.5
   Compiling aho-corasick v0.6.9
   Compiling nom v4.1.1
   Compiling which v2.0.1
   Compiling cexpr v0.3.4
   Compiling env_logger v0.6.0
   Compiling ng-bindgen v0.1.0 (/Users/mf/code/open-source/emacs-ng/rust_src/ng-bindgen)
    Finished release [optimized] target(s) in 31.15s
     Running `/Users/mf/code/open-source/emacs-ng/rust_src/ng-bindgen/target/release/ng-bindgen bindings ../rust_src/crates/emacs/src/bindings.rs`
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 2, kind: NotFound, message: "No such file or directory" }', src/main.rs:223:59
stack backtrace:
   0:        0x101b72b54 - std::backtrace_rs::backtrace::libunwind::trace::h79c24a8108eef51e
                               at /rustc/42816d61ead7e46d462df997958ccfd514f8c21c/library/std/src/../../backtrace/src/backtrace/libunwind.rs:90:5
   1:        0x101b72b54 - std::backtrace_rs::backtrace::trace_unsynchronized::hf491b9388f4887f5
                               at /rustc/42816d61ead7e46d462df997958ccfd514f8c21c/library/std/src/../../backtrace/src/backtrace/mod.rs:66:5
   2:        0x101b72b54 - std::sys_common::backtrace::_print_fmt::h5132bce5284c3ec2
                               at /rustc/42816d61ead7e46d462df997958ccfd514f8c21c/library/std/src/sys_common/backtrace.rs:67:5
   3:        0x101b72b54 - <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::hba4e1e451ca8711d
                               at /rustc/42816d61ead7e46d462df997958ccfd514f8c21c/library/std/src/sys_common/backtrace.rs:46:22
   4:        0x101b9379e - core::fmt::write::h7baaf1618474dae0
                               at /rustc/42816d61ead7e46d462df997958ccfd514f8c21c/library/core/src/fmt/mod.rs:1094:17
   5:        0x101b6f9da - std::io::Write::write_fmt::hd293de47cc154cdf
                               at /rustc/42816d61ead7e46d462df997958ccfd514f8c21c/library/std/src/io/mod.rs:1580:15
   6:        0x101b74aef - std::sys_common::backtrace::_print::hb9d4bc7b9e0ae081
                               at /rustc/42816d61ead7e46d462df997958ccfd514f8c21c/library/std/src/sys_common/backtrace.rs:49:5
   7:        0x101b74aef - std::sys_common::backtrace::print::h82a68481004d7b57
                               at /rustc/42816d61ead7e46d462df997958ccfd514f8c21c/library/std/src/sys_common/backtrace.rs:36:9
   8:        0x101b74aef - std::panicking::default_hook::{{closure}}::h11b9cc5ac5c4d127
                               at /rustc/42816d61ead7e46d462df997958ccfd514f8c21c/library/std/src/panicking.rs:208:50
   9:        0x101b745f9 - std::panicking::default_hook::hfe650a460287c541
                               at /rustc/42816d61ead7e46d462df997958ccfd514f8c21c/library/std/src/panicking.rs:225:9
  10:        0x101b750e5 - std::panicking::rust_panic_with_hook::h5212f5e986dcd234
                               at /rustc/42816d61ead7e46d462df997958ccfd514f8c21c/library/std/src/panicking.rs:591:17
  11:        0x101b74c95 - std::panicking::begin_panic_handler::{{closure}}::hd4a4baba3ac1c064
                               at /rustc/42816d61ead7e46d462df997958ccfd514f8c21c/library/std/src/panicking.rs:497:13
  12:        0x101b72ff8 - std::sys_common::backtrace::__rust_end_short_backtrace::h5a76e76b61bd088d
                               at /rustc/42816d61ead7e46d462df997958ccfd514f8c21c/library/std/src/sys_common/backtrace.rs:141:18
  13:        0x101b74bfa - rust_begin_unwind
                               at /rustc/42816d61ead7e46d462df997958ccfd514f8c21c/library/std/src/panicking.rs:493:5
  14:        0x101ba951f - core::panicking::panic_fmt::h6b7498085d32aaee
                               at /rustc/42816d61ead7e46d462df997958ccfd514f8c21c/library/core/src/panicking.rs:92:14
  15:        0x101ba9605 - core::result::unwrap_failed::h4595b85dafd53acf
                               at /rustc/42816d61ead7e46d462df997958ccfd514f8c21c/library/core/src/result.rs:1355:5
  16:        0x1019cfa27 - ng_bindgen::main::hc66d4bd5326bd528
  17:        0x1019d112a - std::sys_common::backtrace::__rust_begin_short_backtrace::h3b114582aa621a3d
  18:        0x1019d114c - std::rt::lang_start::{{closure}}::h9d2c926287069838
  19:        0x101b75454 - core::ops::function::impls::<impl core::ops::function::FnOnce<A> for &F>::call_once::h88801ec30fa967bc
                               at /rustc/42816d61ead7e46d462df997958ccfd514f8c21c/library/core/src/ops/function.rs:259:13
  20:        0x101b75454 - std::panicking::try::do_call::ha5838b1ed53bb3ce
                               at /rustc/42816d61ead7e46d462df997958ccfd514f8c21c/library/std/src/panicking.rs:379:40
  21:        0x101b75454 - std::panicking::try::h2c2c426e3f3c01a8
                               at /rustc/42816d61ead7e46d462df997958ccfd514f8c21c/library/std/src/panicking.rs:343:19
  22:        0x101b75454 - std::panic::catch_unwind::h383eb7eff10b175f
                               at /rustc/42816d61ead7e46d462df997958ccfd514f8c21c/library/std/src/panic.rs:431:14
  23:        0x101b75454 - std::rt::lang_start_internal::h09b48eb36ffca70d
                               at /rustc/42816d61ead7e46d462df997958ccfd514f8c21c/library/std/src/rt.rs:51:25
  24:        0x1019d0cb9 - _main
make[1]: *** [../rust_src/crates/emacs/src/bindings.rs] Error 101
make: *** [src] Error 2
```

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -65,6 +65,8 @@ jobs:
                     - os: ubuntu-latest
                       build: webrender
                     - os: macos-latest
+                      build: general
+                    - os: macos-latest
                       build: webrender
 
         steps:

--- a/rust_src/crates/emacs/build.rs.in
+++ b/rust_src/crates/emacs/build.rs.in
@@ -222,8 +222,9 @@ fn generate_bindings() {
         if arg.starts_with("-I") {
             // we're running clang from a different directory, so we have to adjust any relative include paths
             let path = Path::new("../src").join(arg.get(2..).unwrap());
-            let buf = std::fs::canonicalize(path).unwrap();
-            processed_args.push(String::from("-I") + &buf.to_string_lossy());
+            if let Ok(buf) = std::fs::canonicalize(path) {
+                processed_args.push(String::from("-I") + &buf.to_string_lossy());
+            }
         } else {
             if !arg.is_empty() && !arg.starts_with("-M") && !arg.ends_with(".d") {
                 processed_args.push(arg.into());

